doc/fundamentals/getting_data_out_of_pachyderm.md

```diff
@@ -0,0 +1,68 @@
+# Getting Data Out of Pachyderm
+
+Once you've got one or more pipelines built and have data flowing through Pachyderm, you need to be able to track that data flowing through your pipeline(s) and get results out of Pachyderm. Let's use the [OpenCV pipeline](../getting_started/beginner_tutorial) as an example. 
+
+Here's what our pipeline and the corresponding data repositories look like:
+
+![alt tag](opencv.jpg)
+
+Every commit of new images into the "images" data repository results in output commits of results into the "edges" data repository. But how do we get our results out of Pachyderm?  Moreover, how would we get the particular result corresponding to a particular input image?  That's what we will explore here.
```

Comment:
should be ..."results in a corresponding output commit in the "edges" data repository."...